ninja-core/src/main/java/ninja/template/TemplateEngineFreemarker.java

```diff
@@ -333,7 +335,23 @@ public String getContentType() {
 
     @Override
     public String getSuffixOfTemplatingEngine() {
-        return FILE_SUFFIX;
+        return fileSuffix;
+    }
+    
+    /**
+     * Will override the default file suffix for freemarker templates.
+     * 
+     * @param fileSuffix file extension
+     */
+    public void setSuffixOfTemplatingEngine(String fileSuffix) {
+        this.fileSuffix = fileSuffix;
+    }
+    
+    /**
+     * @return the freemarker configuration object
+     */
+    public Configuration getConfiguration() {
```

Comment:
When can we fiddle with the Configuration object? Anytime? Or only during startup. Is it threadsafe? Imho this should be documented (in the docs and in the javadoc...).